app/models/knight.rb

```diff
@@ -6,8 +6,19 @@ def puts_piece_type
     puts "This is a Knight"
   end
 
-  def valid_move?
-    # put code for how this piece should move here
+  def legal_move?(x, y)
+    # knight moves two squares in horiz. or vertical then one space in the other of the two directions, whichever one was NOT taken first, so if horiz then vert or if vert then horiz 
+    if x >= 0 and x <= 7 and y >= 0 and y <= 7
```

Comment:
As discussed in stand-up, move this into a separate method, on-board? or something like that.